lib/git.ml

```diff
@@ -164,3 +164,10 @@ let tag tag_name tag_action =
 
 let uncommit () = Process.proc "git reset HEAD~1"
 let unstash () = Process.proc "git stash pop"
+
+let done_ () =
+  let prev_branch = get_current_branch () in
+  let main_branch = fetch_main_branch () in
+  switch None;
+  if String.( <> ) prev_branch main_branch then
+    Process.proc @@ Printf.sprintf "git branch -d %s" prev_branch
```

Comment:
Let's use full flag name 🙂 
Since users don't need to type it, I prefer having full commands in the display so you can easily see what's actually happening. It may be difficult to guess the meaning behind short flags like `-d` or `-s` or `-f`
```suggestion
    Process.proc @@ Printf.sprintf "git branch --delete %s" prev_branch
```